apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js

```diff
@@ -703,7 +703,7 @@ async function openLinksInParentFrame( calypsoPort ) {
 	sidebarsObserver.observe( body, { childList: true } );
 
 	const popoverSlotObserver = new window.MutationObserver( ( mutations ) => {
-		const isComponentsPopover = ( node ) => node.classList.contains( 'components-popover' );
+		const isComponentsPopover = ( node ) => node?.classList.contains( 'components-popover' );
```

Comment:
Ops, this was actually added by accident when I was testing. This is not needed (nor desired) as we already have the `if (! node )` below.